models/request.js

```diff
@@ -0,0 +1,74 @@
+var User = require('./user');
+var Service = require('./service');
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
+      recipient_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      services_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
```

Comment:
This should reflect:
`services_id INT NOT NULL`

`primaryKey` and `autoIncrement` are not needed
not sure what `unique` does